cargo-concordium/Cargo.toml

```diff
@@ -20,6 +20,7 @@ ptree = "0.4"
 strsim = "0.10"
 num-bigint = "0.4.3"
 num-traits = "0.2.15"
+cargo-generate = "0.16.0"
```

Comment:
I don't see why this is needed. Did you want to use `cargo-generate` as a library?